agentops/worker.py

```diff
@@ -21,7 +21,12 @@ def __init__(self, config: Configuration) -> None:
 
     def add_event(self, event: dict) -> None:
         with self.lock:
-            self.queue.append(event)
+            for existing_event in self.queue:
+                if existing_event.get('id') == event.get('id'):
+                    existing_event.update(event)
+                    break
+            else:
```

Comment:
Thx @stateofkate for the PR! I know we're amending it but if we don't: indentation here is off :)

Reply:
Disregard TIL for/else is a thing!